snuba/querylog/factory.py

```diff
@@ -0,0 +1,34 @@
+from snuba.clickhouse.query_dsl.accessors import get_time_range
+from snuba.datasets.dataset import Dataset
+from snuba.datasets.entities.factory import get_entity
+from snuba.datasets.factory import get_dataset_name
+from snuba.query.data_source.projects_finder import ProjectsFinder
+from snuba.query.logical import Query
+from snuba.querylog.query_metadata import SnubaQueryMetadata
+from snuba.request import Request
+from snuba.utils.metrics.timer import Timer
+
+
+def get_snuba_query_metadata(
```

Comment:
Should this be `create_snuba_query_metadata()` instead?

Reply:
done